lib/NetworkManager.js

```diff
@@ -377,6 +384,13 @@ class Request {
     return this._frame;
   }
 
+  /**
+   * @return {!Array<!Request>}
+   */
+  redirectChain() {
+    return this._redirectChain;
```

Comment:
It looks like we are returning a live array that might change after the user receives it. Slice here?